vinyldns/resource_zone_test.go

```diff
@@ -22,26 +22,44 @@ import (
 	"github.com/vinyldns/go-vinyldns/vinyldns"
 )
 
+const (
+	zName             = "system-test."
+	zEmail            = "email@foo.com"
+	zEmailUpdated     = "updated_email@foo.com"
+	zConName          = "vinyldns."
+	zConKey           = "nzisn+4G2ldMn0q1CV3vsg=="
+	zConKeyName       = "vinyldns."
+	zConPrimaryServer = "vinyldns-bind9"
```

Comment:
If I understand the spec correctly, then I believe that a fully qualified domain name, IP address, or IPAddr:Port pair would more clearly communicate the type of data desired. While a simple unqualified host name might work, it seems like an uncommon case.  https://www.vinyldns.io/api/zone-model.html#zone-conn-attr

Reply:
@rhenning Are you referring to `vinyldns-bind9`? If so, I agree, but this is the `docker-compose` service name. If it's not clear, these acceptance tests are running against a `docker-compose`'d VinylDNS setup.

Reply:
@rhenning just the host name OR the IP address is the common case, as 99% of DNS servers talk on port 53.  Actually, having the port specifier is the uncommon case (because you cannot bind to lower ports on most linux machines) and we need to do uncommon things for our tests

Reply:
Got it. I hadn't realized that the tests were running against a docker-compose container. Totally makes sense why it's an unqualified hostname now. Disregard last transmission!